src/targets/gpu/include/migraphx/gpu/context.hpp

```diff
@@ -311,7 +311,6 @@ struct context
         value result;
         result["events"]  = events.size();
         result["streams"] = current_device->nstreams();
-        result["gfx_name"] = get_current_device().get_gfx_name();
```

Comment:
Why is this removed on serialization?

Reply:
I added this earlier when getting FP8 OCP in to query the gfx number from the driver. We could keep it, but it would not be used anywhere anymore.